templates/Collector/db.html.twig

```diff
@@ -252,17 +259,22 @@
                                             </div>
 
                                             {% if query.runnable %}
-                                                <div id="original-query-{{ i }}-{{ loop.parent.loop.index }}" class="sql-runnable hidden">
+                                                <div id="original-query-{{ i }}-{{ loop.parent.loop.index }}" class="sql-runnable">
                                                     {% set runnable_sql = (query.sql ~ ';')|doctrine_replace_query_parameters(query.params) %}
                                                     {{ runnable_sql|doctrine_prettify_sql }}
-                                                    <button class="btn btn-sm hidden" data-clipboard-text="{{ runnable_sql|e('html_attr') }}">Copy</button>
+                                                    <div>
+                                                        <button class="btn btn-sm hidden" data-clipboard-text="{{ runnable_sql|e('html_attr') }}">Copy</button>
+                                                        <a class="btn btn-sm" href="{{ path('_profiler', { panel: 'db', token: token, page: 'run', connection: connection, query: i }) }}" onclick="return run(this);" data-target-id="run-{{ i }}-{{ loop.parent.loop.index }}">Run</a>
+                                                    </div>
                                                 </div>
                                             {% endif %}
 
                                             {% if query.explainable %}
                                                 <div id="explain-{{ i }}-{{ loop.parent.loop.index }}" class="sql-explain"></div>
                                             {% endif %}
 
+                                            <div id="run-{{ i }}-{{ loop.parent.loop.index }}" class="sql-explain"></div>
```

Comment:
shouldn't this be wrapped in `{% if query.runnable %}` condition? Or you believe the previous condition is not necessary?